src/main/resources/schema.sql

```diff
@@ -1,7 +1,18 @@
+DROP TABLE IF EXISTS USER;
 DROP TABLE IF EXISTS BOOKABLE;
 DROP TABLE IF EXISTS LOCATION;
 DROP TABLE IF EXISTS BOOKING;
 
+CREATE TABLE USER (
+    USER_ID            VARCHAR(36) NOT NULL,
+    CONSTRAINT PK_USER PRIMARY KEY (USER_ID),
+    EXTERNAL_USER_ID   VARCHAR(36) NOT NULL,
```

Comment:
I'd bump this to 64 at least (sha-256 length)... not all providers use GUID like microsoft